app/sessions.server.ts

```diff
@@ -64,7 +64,7 @@ export async function getLoggedInUser(request: Request): Promise<User | null> {
 /** Destroy the session cookie  */
 export async function clearCookie(request: Request) {
   let session = await storage.getSession(request.headers.get("Cookie"));
-  return redirect("/", {
+  return json("/", {
```

Comment:
Just curious, why use json instead of redirect?

Reply:
Loaders are executed before the page is rendered in Remix, and this code is used by the loaders in `/logout`. If I used a redirect, the loader would cause a redirect before the clientside log out occurred